__tests__/utils.test.js

```diff
@@ -1,8 +1,21 @@
+const app = require("../app");
+const request = require("supertest");
 const {
   convertTimestampToDate,
   createRef,
   formatComments,
 } = require("../db/seeds/utils");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
```

Comment:
Please separate your endpoint tests into a separate test file! The utils.test file should only house utility functions.